docs/reference/Scheduler.md

```diff
@@ -403,4 +406,13 @@ It is represented as:
 - **name**: Name of the port. Facilitates on recognition;
 - **protocol**: Port protocol. Can be UDP, TCP or SCTP.;
 - **port**: The port exposed.
-- **hostPortRange**: The [port range](#portrange) for the port to be allocated in the host. Mutually exclusive with the port range configured in the root structure.
\ No newline at end of file
+- **hostPortRange**: The [port range](#portrange) for the port to be allocated in the host. Mutually exclusive with the port range configured in the root structure.
+
+#### PDB Max Unavailable
+
+A string value that defines the disruption budget of Game Rooms from a specific scheduler.
+Maestro will create a [PDB Resource](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)
+to prevent evictions drastically impacting availability of the Game Rooms.
+
+By default this value is set to 10%, so at worst runtime can evit 10% of the pods. There is no way to control
```

Comment:
wdyt about adding a ref link to the configuration?

Reply:
Do you mean the kubernetes configuration doc or maestro yaml/env configuration?

Reply:
I've added a document specifying the order of precedence in Maestro configuration, also there's a link to kubernetes